takeshape/penny
# Fix error when building with no review services enabled

### Screenshots

![Screen Shot 2022-11-22 at 1 28 17 PM](https://user-images.githubusercontent.com/211047/203424670-d88d134c-a219-4c64-8c5f-c4d9aa42cd00.png)

### Test Plan (Steps to test):

1. Log into the takeshape project at https://app.takeshape.io/project/06ccc3dc-a9da-4f5b-9142-5a104db52ee3/
1. Create a branch
1. On your branch, remove the authentication values for the `reviews-io` and `trustpilot` services
1. Pull down this PR and change `NEXT_PUBLIC_TAKESHAPE_API_URL` in `.env.local` to your branch's api url
1. Edit `src/config/reviews` and set `enableReviewsIo` to `false`
1. `npm run dev`
1. Check out the site, make sure nothing breaks
1. Try enabling the review services too and make sure they still work (for each you need to add the authentication value back in and edit `src/config/reviews` to enable

### Checklist:
